xbmc/epg/Epg.cpp

```diff
@@ -52,9 +50,7 @@ CEpg::CEpg(int iEpgID, const CStdString &strName /* = "" */, const CStdString &s
   m_pvrChannel = empty;
 }
 
-CEpg::CEpg(CPVRChannelPtr channel, bool bLoadedFromDb /* = false */) :
-    m_bChanged(!bLoadedFromDb),
-    m_bTagsChanged(false),
+CEpg::CEpg(CPVRChannelPtr channel) :
```

Comment:
I tend to disagree. Shared pointers are used to avoid copies of the actual data. This is in no way related to the C++ concept of passing parameters as references. If the shared_ptr object is passed by reference this is more efficient compared to by value as no copy of the object is needed. Why should a copy of the shared_ptr instance (and thus incrementing refcount) be needed (or "better") here?